cmd/ocm-backplane/login/login.go

```diff
@@ -93,6 +101,9 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 			return err
 		}
 		logger.Debugf("Using backplane Proxy URL: %s\n", proxyUrl)
+	} else {
+
+		proxyUrl = bpConfig.ProxyURL
```

Comment:
Isn't this already being done in the next line?
i.e; `if len(proxyUrl) == 0` (which will be the case if `--proxy` isn't specified) then set the bpConfig's `ProxyURL` as `proxyUrl`.

Reply:
yeah .. You are right.